app/_src/production/secure-deployment/certificates.md

```diff
@@ -201,21 +206,21 @@ KUMA_API_SERVER_HTTPS_TLS_CERT_FILE=/tmp/tls.crt \
 {% endtab %}
 {% endtabs %}
 
-  If you recieve an error like the following, make sure you are using a supported certificate type and that the certificate doesn't contain incomplete or corrupted data:
+  If you receive an error like the following, make sure you are using a supported PEM certificate and that the certificate doesn't contain incomplete or corrupted data:
   ```sh
   Warning  FailedCreate  3m39s (x18 over 14m)  replicaset-controller  Error creating: Internal error occurred: failed calling webhook "namespace-kuma-injector.kuma.io": could not get REST client: unable to load root certificates: unable to parse bytes as PEM block
   ```
 
 
-3. Configure secure connection using `kumactl` CLI tool:
+3) Configure secure connection using the `kumactl` command-line tool:
```

Comment:
I think you could use CLI but we need to define it here https://github.com/kumahq/kuma-website/blob/2359791dce1109e75a4dbf3fe1f1f65e98a5a78e/.github/styles/Vocab/Base/accept.txt#L19